### DISCUSSION TITLE
Which version of the engine is used by the game?

### DISCUSSION THREAD

**Discussion by Wol1Fram** [Libre-TrainSim/Libre-TrainSim#515] [Q&A] [answered]
The other day I decided to try myself in developing a railway simulator and for a long time I chose the engine, after which I came across a vidos in a YouTube with this game and became interested in it, decided to try to create something based on this simulator

**Comment by nalquas** [comment, marked as answer]
Libre-TrainSim currently uses Godot 3. The most up-to-date version from the Godot-3-branch (currently 3.5.2, see [Godot's website](https://godotengine.org/download/3.x/)) should work.
